TemplateApplication/Schedule/EventView.swift

```diff
@@ -26,7 +26,7 @@ struct EventView: View {
                     return // user cancelled the task
                 }
 
-                event.complete()
+                _ = try? event.complete()
```

Comment:
@lukaskollmer Wondering if that is the best way to handle any errors here in the template app or if we should showcase a better recovery/error surfacing mechanism?

Reply:
The expectation is that this doesn’t throw (as the button would have been disabled if the policy forbid it). Though the user could have left the questionnaire open for several hours (or across midnight), in this case this might throw and we should handle this I guess? E.g., showing an error alert suggesting to try again and dismissing the sheet. We should definitely make sure that we not Save the results if completion fails.